playbooks/files/create_cce_cluster.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import sys
+import time
+
+import openstack
+from otcextensions import sdk
+
+openstack.enable_logging(True, http_debug=True)
+
+# An 'otc' is a cloud connection with name 'otc' configured in the clouds.yaml
+conn = openstack.connect()
+
+# Register OTC Extensions
+sdk.register_otc_extensions(conn)
+
+cluster = conn.cce.find_cluster(sys.argv[1])
+dict = {
+    'metadata': {
+        'name': sys.argv[1]
+    },
+    'spec': {
+        'type': 'VirtualMachine',
+        'version': 'v1.11.7-r2',
+        'hostNetwork': {
+            'vpc': sys.argv[2],
```

Comment:
vpc is currently renamed to router_id

Reply:
I oriented my self @ the official docu. After now how this mapping is done, I see the point. Thanks for the hint. Changed it.

Reply:
Ich changed both values, but get "BadRequest" if I do this. Switching back works again.
```
(py36) ubuntu@tino-dev2:~/git/apimon-tests/playbooks/files$ python create_cce_cluster.py tino-cce1 26ca2783-dc40-4e3a-95b1-5a0756441e12 25d24fc8-d019-4a34-9fff-0a09fde6a9cb
Traceback (most recent call last):
  File "create_cce_cluster.py", line 37, in <module>
    cluster = conn.cce.create_cluster(**dict)
  File "/home/ubuntu/git/python-otcextensions/otcextensions/sdk/cce/v3/_proxy.py", line 58, in create_cluster
    _cluster.Cluster, prepend_key=False, **attrs
  File "/home/ubuntu/git/python-otcextensions/.tox/py36/lib/python3.6/site-packages/openstack/proxy.py", line 417, in _create
    return res.create(self, base_path=base_path)
  File "/home/ubuntu/git/python-otcextensions/otcextensions/sdk/cce/v3/_base.py", line 93, in create
    return super(Resource, self).create(session, prepend_key, base_path)
  File "/home/ubuntu/git/python-otcextensions/.tox/py36/lib/python3.6/site-packages/openstack/resource.py", line 1278, in create
    self._translate_response(response, has_body=has_body)
  File "/home/ubuntu/git/python-otcextensions/.tox/py36/lib/python3.6/site-packages/openstack/resource.py", line 1107, in _translate_response
    exceptions.raise_from_response(response, error_message=error_message)
  File "/home/ubuntu/git/python-otcextensions/.tox/py36/lib/python3.6/site-packages/openstack/exceptions.py", line 229, in raise_from_response
    http_status=http_status, request_id=request_id
openstack.exceptions.BadRequestException: BadRequestException: 400: Client Error for url: https://cce.eu-de.otc.t-systems.com/api/v3/projects/16d53a84a13b49529d2e2c3646691288/clusters, Bad Request
```